content/fast/minify-css/index.md

```diff
@@ -55,104 +55,115 @@ In practice, you would like to introduce this technique as one of the steps of y
+<strong>Note:</strong> On the previous <strong>Lighthouse</strong> report, the only file that appears as an opportunity for minification is <strong>animate.css</strong>. The reason is that the bytes saved after minifyng <strong>style.css</strong> would be under the threshold that the tool considers "significant".
+In any case, recall that minification is a general best practice, which sould be applied to any CSS file, regardless that they show up or not as suggestions on performance tools.
 </div>
 
 ## CSS Minification with Webpack:
 
-The way the CSS files are produced works as follows:
-
-- **index.js** imports **src/styles.css**, meaning the resulting JS bundle will contain the content of the CSS file inlined.
-- In **webpack-config.js** [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin) is used, so, after the previous step, the CSS can be extracted to its own file.
-
-Take a look at the content of these files:
+Before jumping into the optimizations, take some time understanding how the [demo site](https://glitch.com/edit/#!/fav-kitties-animated?path=webpack.config.js:1:0]) works:
 
 <div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
   <iframe
     allow="geolocation; microphone; camera; midi; vr; encrypted-media"
-    src="https://glitch.com/embed/#!/embed/minify-css-unoptimized?path=src/index.js&previewSize=0"
-    alt="minify-css-unoptimized on Glitch"
+    src="https://glitch.com/embed/#!/embed/fav-kitties-animated?path=webpack.config.js&previewSize=0"
+    alt="fav-kitties-animated on Glitch"
     style="height: 100%; width: 100%; border: 0;">
   </iframe>
 </div>
 
+By default, the resulting JS bundle that Webpack produces would contain the content of the CSS files inlined. Since we want each CSS to be on its own file, we are using two complementary plugins:
+
+- [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin) will extract each CSS into its own file, as one o the steps of the build process.
+- [webpack-fix-style-only-entries](https://github.com/fqborges/webpack-fix-style-only-entries) is used to correct an issue on Wepback 4, to avoid generating an extra JS file for each CSS declared as entry in **webpack-config.js**.
+
 You will now make some changes in the project:
 
-1. Open the project [in Glitch](https://glitch.com/~minify-css-unoptimized).
-1. Click “View Source”, and then, “Remix and Edit”, to make changes on the project.
-1. Click the “Tools” optoin in the lower left corner of the edit view, and go to “Console”.
+1. Open the project [in Glitch](https://glitch.com/~fav-kitties-animated).
+1. Click “View Source”, and then, click on the name of the project in the upper left side.
+1. In the drop-down menu, choose “Remix and Edit” 🎤, to make changes on the project.
+1. Once in the cloned project, go to the “Tools” option in the lower left corner of the edit view, and click on “Console”.
```

Comment:
For consistency: "go to the Tools ..." => "click **Tools** (in the lower-left corner of the edit view), then select **Console**.

Reply:
_maybe_ add "The Glitch console opens in a new browser tab."